scripts/rebuild.sh

```diff
@@ -3,9 +3,15 @@
 set -e
 
 cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")"
-if [ ! -e ../vcpkg ]
-then
-    vcpkg_commit=d765306b074717dea8dc1c4723e1b025acb61c2d
+if [ ! -e ../vcpkg ]; then
+    vcpkg_commit=d5b03c125afee1d9cef38f4cfa77e229400fb48a
+    filename="commit.txt"
+    if [ -f "$filename" ]; then
+        content=$(cat commit.txt)
+        if [ "$vcpkg_commit" != "$content" ]; then
+            vcpkg_commit=$content
+        fi
+    fi
```

Comment:
```suggestion
    vcpkg_commit=$(cat commit.txt)
    if [ ! -e commit.txt ]; then; exit 1; fi
```

I don't see any reason why we need to have a separate commit here.

Reply:
Incorporated the suggestions.